docs/jumpstart/week-3.md

```diff
@@ -0,0 +1,418 @@
+```
+
+
+### Booleans
+
+Booleans are either the value `false` or `true`. You will also hear the term *condition* which refers to a statement that evaluates to a boolean.
+
+#### Operations
+
+Javascript 2 operators for combining conditions: or `||` & and `&&`. 'or' will return true if either condition is true, and 'and' will only return true only if both conditions are true.
+
+You can also reverse a condition with the not operator `!`.
+
+```js
+const orOne = false || true;                  // This is true
+const orTwo = false || false || false;        // This is false
+const andOne = false && true && true;         // This is false
+const andTwo = true && true;                  // This is true
+const not = !true                             // This is false
+const combination = !true || !false && true;  // This is true
+```
+
+An important concept to be aware of when working with logical operators is the idea of *short-circuiting*. Because 'or' only needs one true condition to be true and 'and' only needs one false to be false, as soon as Javascript encounters one of those values during those operations it'll stop evaluating the rest of the conditions. While this doesn't matter when your conditions are all booleans, once we start using conditions that have *side effects* this behavior will often be exploited.
+
+
+### Arrays
+
+Arrays in Javascript are ordered collections of data.
+
+Arrays are wrapped in box brackets `[ ... ]`. Arrays can hold any kind of data including other arrays.
+
+```js
+const ourArray = ["A first value", 11, ["Nested Value", 0.12], 'A last value'];
+```
+
+#### Operations
+
+You can refer to values in arrays by their *index*. Arrays in Javascript are *0-indexed* meaning the first value in the array has an index of 0, the next value has an index of 1, and so on until the last index that is the length of the array minus 1.
+
+You can also get the length of an array with the `.length` property, identical to a string.
+
+```js
+const arr = [1, 2, 3, 4];
+const arrLength = arr.length; // Equal to 4
+const secondEl = arr[1]; // Equal to 2
+const thirdEl = arr[arr.length - 1]; // Equal to 4
+```
+
+Arrays are also modifiable. You can assign a value to a particular index in an array with the following syntax:
+
+```js
+const arr = [1, 2, 3, 4];
+const beginning = arr[0]; // Equal to 1
+arr[0] = 100;
+const newBeginning = arr[0]; // Equal to 100
+```
+
+Arrays also define several built in functions. You can read a full list of them on [MDN here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).
+
+### Objects
+
+Objects in Javascript are used to hold more complex data structures.
+
+Objects are wrapped in curly brackets `{ ... }`. Within the brackets, you can define a list of properties. A property has a key, which is just a single unquoted word, and a value which can be any other Javascript data type (including another object!).
+
+```js
+const ourObject = {
+    aKey: "A String Value",
```

Comment:
Should we show multi-word keys with single quotes?

Reply:
Generally it's not great practice to have multi word keys so I didn't include it